CondTools/RunInfo/interface/LHCInfoPerLSPopConSourceHandler.h

```diff
@@ -0,0 +1,65 @@
+#ifndef LHCINFOPOPCONSOURCEHANDLER_H
+#define LHCINFOPOPCONSOURCEHANDLER_H
```

Comment:
same header comment

Reply:
Hi @tvami,
Never considered adding `SortIncludes: true` to [.clang-format](https://github.com/cms-sw/cmssw/blob/master/.clang-format)?

Reply:
That's beyond me, @smuzaffar is there a reason for this set to `false` in default?

Reply:
Of course, that would require a _lot_ of 
```cpp
// clang-format off
...
// clang-format on
```
everywhere...

Reply:
I do not remember the exact reasoning behind explicitly setting `SortIncludes: false` (may be @davidlange6 remembers? ) but my quess is this was done to avoid re-formatting all of cmssw files.

Reply:
it is a change that can reveal code bugs.. Likely no one had the ambition to fix all the bugs that code format change would uncover. But I guess its a change many would like to have done..